### PR TITLE
Add io.gitlab.elescoute.spacelaunch

### PR DESCRIPTION
**Pull request by elescoute**
### Description ###

This is a migration of org.emilien.SpaceLaunch.

Space Launch is a native unofficial SpaceLaunchNow.me client for GNOME, written in Vala and GTK4. It allows to keep track of upcoming rocket launches.

### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by elescoute** [comment]
bot, build io.gitlab.elescoute.spacelaunch

File: README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<img src="https://flathub.org/assets/badges/flathub-badge-en.png" width="120"/></a>
+
+## Dependencies
+
+Please make sure you have these dependencies first before building.
+
+```
+gtk4
+libadwaita-1
+libsoup-2.4
```
Comment (bbhtt) [inline README.md:35]: This is not in runtime, use it from here https://github.com/flathub/shared-modules or better upgrade to libsoup 3

Comment (elescoute) [reply README.md:35]: It's an error. I don't use libsoup anymore. I will correct this and the other errors.

**Comment by elescoute** [comment]
bot, build io.gitlab.elescoute.spacelaunch

**Comment by elescoute** [comment]
Tested succesfully!

**Comment by hfiguiere** [comment]
The other app will need to be EOL rebased after this.

**Comment by elescoute** [comment]
Yes, it's planned.

**Pull request closed without merging**